src/generated/resources/openapi.json

```diff
@@ -1426,6 +1430,26 @@
           }
         }
       },
+      "ScramCredentials" : {
+        "description" : "Scram authentication credentials",
+        "required" : [ "hash_algorithm", "username", "password" ],
```

Comment:
May I trouble you to regenerate the OpenAPI spec, e.g., by starting the sidecar in dev mode? These field names are outdated.